frontend/next.config.js

```diff
@@ -10,7 +10,7 @@ const ContentSecurityPolicy = `
 	script-src 'self' https://app.posthog.com https://js.stripe.com https://api.stripe.com https://widget.intercom.io https://js.intercomcdn.com 'unsafe-inline' 'unsafe-eval';
 	style-src 'self' https://rsms.me 'unsafe-inline';
 	child-src https://api.stripe.com;
-	frame-src https://js.stripe.com/ https://api.stripe.com;
+	frame-src https://js.stripe.com/ https://api.stripe.com https://www.youtube.com/;
```

Comment:
Is this because we will be serving a youtube video?

Reply:
yeap, is that an issue?